fluent/test/fixtures_structure/attribute_with_empty_pattern.json

```diff
@@ -0,0 +1,43 @@
+{
+  "key1": {
+    "val": "Value 1",
+    "attrs": {
+      "attr": {
+        "val": null
```

Comment:
These are all syntax errors in the tooling parser. In the runtime parser I chose to parse them as `null` values, at least for now, to reduce the complexity of this PR and to keep the performance benefits.